zipkin/index.js

```diff
@@ -0,0 +1,137 @@
+
+var Request = require('request')
```

Comment:
I would recommend using wreck instead from the hapi eco-system - it's *much* faster, and more consistent with Seneca in general (Seneca leverages hapi, a lot)
https://github.com/hapijs/wreck